client/sbom.go

```diff
@@ -0,0 +1,112 @@
+package client
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/kondukto-io/kdt/klog"
+	"github.com/spf13/viper"
+)
+
+func (c *Client) ImportSBOM(file string, repo string, form ImportForm) error {
+	klog.Debugf("importing sbom content using file:%s", file)
+
+	projectId := form["project"]
+
+	if projectId != "" {
+		projects, err := c.ListProjects(form["project"], repo)
+		if err != nil {
+			return err
+		}
+
+		if len(projects) == 1 {
+			projectId = projects[0].ID
+			form["project"] = projects[0].Name
+		}
+
+		if len(projects) > 1 {
+			return errors.New("multiple projects found for given parameters")
+		}
+	} else if repo != "" {
```

Comment:
Redundant `} else if repo != "" {` usage instead `} else {`

``` go
} else {
```
is enough. Because at least one of the projectId or repo must be not empty string. So if the projectID == "" then the repo is definitely not empty..

Reply:
Fixed.